fast_agave/blueprints/rest_api.py

```diff
@@ -74,6 +74,21 @@ def wrapper_resource_class(cls):
                 route = self.post(path)
                 route(cls.create)
 
+            """ POST /resource/{id}
+            Create a FastApi endpoint using the method "upload".
+            Enables a POST method using a streaming multipart parer to
+            receive files as form data.
+            """
+            if hasattr(cls, 'upload'):
+
+                @self.post(path + '/{id}')
```

Comment:
esto va a romper la forma en que usamos `POST`. Si deseas asignar a que `id` pertenece ese archivo este dato debe pasarse como datos de form, ya que no se pueden enviar un body con json. 

https://fastapi.tiangolo.com/tutorial/request-forms-and-files/#import-file-and-form

Reply:
tal vez ni sea necesario mandarlo como datos de form. podrías usar `app.current_user_id` para asignar algún valor

Reply:
lo primero estoy de acuerdo, pero lo segundo no creo que aplique porque en el caso de `omni`, necesitamos que el `user_id` sea el del perfil del cliente, no del usuario de `omni`

Reply:
tienes razón, lo segundo es más lógica de negocio. eso no aplica aquí